### ISSUE TITLE
Polling Tango for Solutions

### ISSUE THREAD

**Issue by ifjorissen** [ifjorissen/vrfy#186]
When a user submits a solution (or when a grader / prof reassesses a solution) we poll tango consistently with a python function (see _get_problemresult in course/views.py and the reassess method in course/admin.py). This is probably a bad idea because it occupies one of our gunicorn workers (we only have ~4) during this time period. This can lead to issues where we are getting requests and the number of apache threads created by those requests outnumbers the available gunicorn workers and then we get a 503. 

TL;DR, we should probably do our polling on the client-side w/ javascript where we poll for a 'yes, it's ready' or 'no, not yet' response immediately (that part can be done as a python request) and then to poll again rather than to have the worker enter a function with a sleep timer.

**Comment by ifjorissen** [comment]
If we can get the job queue thing to work (w/ Celery, see #189 ) we shouldn't have as many issues polling and we'll likely have to refactor student submission and reassess and that might make this issue irrelevant

**Comment by ifjorissen** [comment]
closed with #199 and #200
